scripts/versions.sh

```diff
@@ -2,9 +2,24 @@
 # Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
 # See the file LICENSE for licensing terms.
 
-# Set up the versions to be used
-AWM_RELAYER_VERSION=${AWM_RELAYER_VERSION:-'v0.2.2'}
-SUBNET_EVM_VERSION=${SUBNET_EVM_VERSION:-'v0.5.8'}
+# The go version for this project is set from a combination of major.minor from go.mod and the patch version set here.
+GO_PATCH_VERSION=10
```

Comment:
why do we set a go patch version of 10 here?

Reply:
Could you clarify what you mean?

Reply:
nvm I misunderstood it